research/src/pages/light-dismiss/light-dismiss.research.mdx

```diff
@@ -0,0 +1,12 @@
+---
+name: Checkbox Concepts
```

Comment:
Should be Light Dismiss

Reply:
Oh, I didn't update the file when copy pasting. Sorry about that!